inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/project/wizard/KnowledgeBaseCreationWizard.java

```diff
@@ -206,19 +243,93 @@ public IDynamicWizardStep next() {
     private final class LocalRepositoryStep extends DynamicWizardStep {
 
         private static final long serialVersionUID = 8212277960059805657L;
+        private static final String TMP_FILE_NAME = "inception_tmp_kb";
         
         private CompoundPropertyModel<KnowledgeBaseWrapper> model;
         private FileUploadField fileUpload;
+        private WebMarkupContainer listViewContainer;
+        private KnowledgeBaseProfile selectedKnowledgeBaseProfile;
         private boolean completed;
 
         public LocalRepositoryStep(IDynamicWizardStep previousStep,
                 CompoundPropertyModel<KnowledgeBaseWrapper> model) {
             super(previousStep);
             this.model = model;
             completed = true;
-            
+
             fileUpload = new FileUploadField("upload");
             add(fileUpload);
+
+            List<KnowledgeBaseProfile> downloadableKBs = knowledgeBaseProfiles.values().stream()
+                .filter(kb -> RepositoryType.LOCAL.equals(kb.getType()))
+                .collect(Collectors.toList());
+
+            listViewContainer = new WebMarkupContainer("listViewContainer");
+            ListView<KnowledgeBaseProfile> suggestions = new ListView<KnowledgeBaseProfile>(
+                "downloadableKBs", downloadableKBs)
+            {
+                @Override protected void populateItem(ListItem<KnowledgeBaseProfile> item)
+                {
+                    LambdaAjaxLink link = new LambdaAjaxLink("suggestionLink", t -> {
+                        selectedKnowledgeBaseProfile = item.getModelObject();
+                    })
+                    {
+                        @Override protected void onConfigure()
+                        {
+                            // Can not download the same KB more than once
+                            setEnabled(
+                                !downloadedProfiles.containsKey(item.getModelObject().getName()));
+                        }
+                    };
+
+                    String itemLabel = item.getModelObject().getName();
+                    // Adjust label to indicate whether the KB has already been downloaded
+                    if (downloadedProfiles.containsKey(item.getModelObject().getName())) {
```

Comment:
@MdeBoer95  We can add the ontology in the classpath and it will not be needed to download them into tmp folder. WDYS

Reply:
Downloaded ontologies should to into a temporary file which is deleted after the import (mind it should also be deleted in case the import fails, so the delete needs to happen in a "finally" block!